recipe-box-backend/src/test/java/com/endava/recipebox/RecipeBoxBackendApplicationTests.java

```diff
@@ -37,5 +48,43 @@ public void shouldSaveAndReturnUserIfSuccessful() throws Exception {
         this.mockMvc.perform(get("/login"))
                 .andExpect(status().isBadRequest());
     }
+
+    @Test
```

Comment:
Very good job, that you try to add unit tests. But as best practice is to define a test class for each tested class, so in this case you try to test RecipeController class, so let Intellij to help you with creating a new test class for you as follow: click on the RecipeController class, press alt+enter and click on create test class. In this way will be autogenerated the directory and file for you in /test